src/averageAges.js

```diff
@@ -55,7 +78,41 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const findMother = (group, person) => {
+    const mother = group.find(mthr => mthr.name === person.mother);
```

Comment:
Well, it will be more correct here to use something like `person` instead of mother and `potentialChild` (or something like that) instead of `person` 

But best option is destructuring